var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -178,11 +179,13 @@ class Openmpi(AutotoolsPackage):
     patch('configure.patch', when="@1.10.1")
     patch('fix_multidef_pmi_class.patch', when="@2.0.0:2.0.1")
 
+    fabrics = ('psm', 'psm2', 'verbs', 'mxm', 'ucx', 'libfabric')
+
     variant(
         'fabrics',
         default=None if _verbs_dir() is None else 'verbs',
-        description='List of fabrics that are enabled',
-        values=('psm', 'psm2', 'verbs', 'mxm'),
+        description=("List of fabrics that are enabled"),
```

Comment:
Why the parentheses?

Reply:
I really don't remember why I did that. Removed